cypress/tests/integration/signals-analytics/analytics-report/reportPage.spec.js

```diff
@@ -100,6 +100,41 @@ if (IS_HIBANA_ENABLED) {
       });
     });
 
+    it('should properly control the expandables', () => {
+      commonBeforeSteps();
+      cy.visit(PAGE_URL);
+      cy.findByRole('button', { name: 'Add Metrics' }).click();
+      cy.withinDrawer(() => {
+        cy.findByDataId('expandable-content')
+          .eq(0)
+          .should('be.visible');
+        cy.findByDataId('expandable-toggle')
+          .eq(0)
+          .click();
+        cy.findByDataId('expandable-content')
+          .eq(0)
+          .should('not.be.visible');
+        cy.findByDataId('expandable-content')
+          .eq(1)
+          .should('be.visible');
+        cy.findByDataId('expandable-toggle')
+          .eq(1)
+          .click();
+        cy.findByDataId('expandable-content')
+          .eq(1)
+          .should('not.be.visible');
+        cy.findByDataId('expandable-content')
+          .eq(2)
+          .should('be.visible');
+        cy.findByDataId('expandable-toggle')
+          .eq(2)
+          .click();
+        cy.findByDataId('expandable-content')
+          .eq(2)
+          .should('not.be.visible');
+      });
+    });
```

Comment:
To make things a little less brittle, you can use `cy.findByRole` to grab the clickable elements:
```suggestion
it('should properly control the expandables', () => {
  commonBeforeSteps();
  cy.visit(PAGE_URL);
  cy.findByRole('button', { name: 'Add Metrics' }).click();
  cy.withinDrawer(() => {
    cy.findByDataId('expandable-content')
      .eq(0)
      .should('be.visible');
    cy.findByRole('button', { name: /Injection Metrics/g }).click();
    cy.findByDataId('expandable-content')
      .eq(0)
      .should('not.be.visible');
    cy.findByDataId('expandable-content')
      .eq(1)
      .should('be.visible');
   cy.findByRole('button', { name: /Delivery Metrics/g }).click();
   cy.findByDataId('expandable-content')
     .eq(1)
     .should('not.be.visible');
   cy.findByDataId('expandable-content')
     .eq(2)
     .should('be.visible');
    cy.findByRole('button', { name: /Engagement Metrics/g }).click();
    cy.findByDataId('expandable-content')
      .eq(2)
      .should('not.be.visible');
  });
});
```